### DISCUSSION TITLE
preferred orientation

### DISCUSSION THREAD

**Discussion by godbnc** [OrcaSlicer/OrcaSlicer#1508] [General]
![Untitled](https://github.com/SoftFever/OrcaSlicer/assets/64687621/f7fa6f88-5b66-45a5-ad65-16d38fa3e702)
introduce preferred orientation similar to superslicer and add procent to line width :) and zoom for keyboard

**Comment by CodyPrince** [comment]
I'm all for the preferred orientation. I always had superslicer position parts at a 45 and seams set to rear. Kept most of my seams on corners.
